.github/workflows/ci.yml

```diff
@@ -39,6 +43,11 @@ jobs:
         name: Test
         with:
           command: test
+      - uses: actions-rs/cargo@v1
+        name: Format
+        with:
+          command: fmt
+          args: --all -- --check
```

Comment:
This should probably be moved to the same job as clippy and be ran first

Reply:
Done